pyproject.toml

```diff
@@ -170,13 +170,13 @@ filterwarnings = [
 addopts = "-ra"
 
 [tool.cibuildwheel]
-build = "cp311-* cp312-* cp313-*"
```

Comment:
for posterity: the reason I'm able to remove this line is that pypy wheels are now disabled by default, so I can leave it to cibuildwheel to discover what range should be built and tested purely from `requires-python` + what's currently stable.